recipes/control/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "control" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "18ce26206e40f4ba62b2a0bf86943598a6ec3f957e15bd46d17b9e1e314f19d6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    # NOTE : Once Slycot is available as a Windows conda package, update this.
+    - slycot  # [unix]
+
+test:
+  imports:
+    - control
+
+about:
+  home: http://python-control.sourceforge.net
+  license: BSD-3-Clause
```

Comment:
I don't even see a license in the source tree at all: https://github.com/python-control/python-control/tree/0.7.0